rules/modules.go

```diff
@@ -0,0 +1,92 @@
+package rules
+
+import (
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/hcl/v2"
+	"github.com/hashicorp/hcl/v2/hclsyntax"
+	"github.com/terraform-linters/tflint-plugin-sdk/tflint"
+)
+
+var _ tflint.Rule = new(ModulesRule)
+
+type ModulesRule struct {
```

Comment:
Could we rename it to `ModuleSourceRule`?

Reply:
Updated